cmd/expiration-mailer/main.go

```diff
@@ -286,9 +289,35 @@ func (m *mailer) findExpiringCertificates() error {
 			)
 		}
 		if err != nil {
-			m.log.AuditErr(fmt.Sprintf("expiration-mailer: Error loading certificates: %s", err))
-			return err // fatal
+			m.log.AuditErr(
+				fmt.Sprintf(
+					"expiration-mailer: Error loading certificate statuses: %s", err))
+			return err
+		}
+
+		// Now we can sequentially retrieve the certificate details for each of the
+		// certificate status rows
+		var certs []core.Certificate
+		for _, cs := range statuses {
+			var cert core.Certificate
+			err := m.dbMap.SelectOne(&cert,
+				`SELECT
+				cert.*
+				FROM certificates AS cert
+				WHERE serial = :serial`,
+				map[string]interface{}{
+					"serial": cs.Serial,
+				},
+			)
+			if err != nil {
+				m.log.AuditErr(
+					fmt.Sprintf(
+						"expiration-mailer: Error loading cert %q : %s", cert.Serial, err))
+				return err // fatal
```

Comment:
This was a comment already in the mailer. I'd rather remove it because I'm not sure what the original author intended (if it was me I've definitely forgotten!)